index.js

```diff
@@ -59,13 +60,22 @@ module.exports = function(source) {
     }
   });
 
+  manualImports = manualImports.map(function (it) { return utils.lookup(it, options.paths, '', true); });
```

Comment:
Can you make some unit tests?

I'm trying to think through how you are using `utils.lookup`. Maybe we need to add a resolver for PatchCache but I'm not convinced that's the solution yet. But we definitely don't want this manual solution. It supports your use case but not most of the use cases for this.